packages/account/src/account.ts

```diff
@@ -620,12 +620,46 @@ export class Account extends AbstractAccount {
     if (estimateTxDependencies) {
       await this.provider.estimateTxDependencies(transactionRequest);
     }
+
     return this.provider.sendTransaction(transactionRequest, {
       awaitExecution,
       estimateTxDependencies: false,
     });
   }
 
+  /**
+   * Sends a multiple transactions to the network.
+   *
+   * @param transactionRequestLike - An array of transaction requests to be sent.
+   * @param sendMultipleTransactionsParams - The provider send transaction parameters (optional).
+   */
+  async sendMultipleTransactions(
+    transactionRequestLike: TransactionRequestLike[],
+    { estimateTxDependencies = true, awaitExecution }: ProviderSendTxParams = {}
+  ) {
+    const transactionsRequest = transactionRequestLike.map(transactionRequestify);
+
+    if (estimateTxDependencies) {
+      await this.provider.estimateMultipleTxDependencies(transactionsRequest);
+    }
+
+    for (const tx of transactionsRequest) {
+      const response = await this.provider.sendTransaction(tx, {
+        awaitExecution,
+        estimateTxDependencies: false,
+      });
+
+      if (!response) {
+        throw new FuelError(
+          ErrorCode.TRANSACTION_FAILED,
+          'Transaction failed during batch processing. Aborting further transactions.'
+        );
+      }
+    }
+
+    // TODO: decide whether we should return an array of successful transactions or non
```

Comment:
@Torres-ssf, if a X was processed, but X + 1 wasn't processed, is it possible to revert X?

Reply:
@mvares No, we cannot revert a TX that was already processed. 

For this one needs to return an array of successful/reverted transactions